app/models/vote.rb

```diff
@@ -0,0 +1,48 @@
+class Vote < ActiveRecord::Base
```

Comment:
Good work refactoring a number of small one line methods onto your Vote class that allow the vote class to deal with maintaining and answering questions about its state!